h2o-genmodel/src/main/java/hex/genmodel/tools/PredictCsv.java

```diff
@@ -188,7 +192,9 @@ private void run() throws Exception {
         }
 
         output.write("\n");
+        lineNum++;
       }
+      reader.close();
```

Comment:
why did you move this inside the try block? close() should be called in finally, or better use the try(with) syntax.